products/kolibri.md

```diff
@@ -0,0 +1,168 @@
+---
+title: KoliBri (Public UI)
+addedAt: 2025-12-19
+category: framework
+tags: web-components accessibility javascript-runtime typescript government bitv wcag a11y
+iconSlug: kolibri
+shortDescription: KoliBri is an accessible Web Component Library
+permalink: /kolibri
+alternate_urls:
+  - /public-ui
+  - /public-ui-kolibri
+versionCommand: npm list @public-ui/components
+releasePolicyLink: https://github.com/public-ui/kolibri/releases
+changelogTemplate: https://github.com/public-ui/kolibri/releases/tag/latest
+eoasColumn: Active Development
+eolColumn: Security Support
+
+identifiers:
+  - purl: pkg:github/public-ui/kolibri
+  - purl: pkg:npm/%40public-ui/components
+  - purl: pkg:npm/%40public-ui/angular-v19
+  - purl: pkg:npm/%40public-ui/angular-v20
+  - purl: pkg:npm/%40public-ui/angular-v21
+  - purl: pkg:npm/%40public-ui/hydrate
+  - purl: pkg:npm/%40public-ui/kolibri-cli
+  - purl: pkg:npm/%40public-ui/mcp
+  - purl: pkg:npm/%40public-ui/react
+  - purl: pkg:npm/%40public-ui/react-v19
+  - purl: pkg:npm/%40public-ui/preact
+  - purl: pkg:npm/%40public-ui/sample-react
+  - purl: pkg:npm/%40public-ui/solid
+  - purl: pkg:npm/%40public-ui/svelte
+  - purl: pkg:npm/%40public-ui/theme-bwst
+  - purl: pkg:npm/%40public-ui/theme-default
+  - purl: pkg:npm/%40public-ui/theme-ecl
+  - purl: pkg:npm/%40public-ui/visual-tests
+  - purl: pkg:npm/%40public-ui/vue
+
+auto:
+  methods:
+    - npm: "@public-ui/components"
+
+releases:
+  - releaseCycle: "5"
+    releaseDate: 2026-12-01
+    eoas: false
+    eol: 2027-12-31
+    latest: "5.0.0"
+    latestReleaseDate: 2026-12-01
+    supportedPhpVersions: STS
+
+  - releaseCycle: "4"
+    releaseDate: 2025-12-01
+    eoas: false
+    eol: 2028-12-31
+    latest: "4.0.0"
+    latestReleaseDate: 2025-12-23
+    lts: true
+
+  - releaseCycle: "3"
+    releaseDate: 2024-12-01
+    eoas: 2025-12-23
+    eol: 2025-12-31
+    latest: "3.0.9"
+    latestReleaseDate: 2025-12-18
+
+  - releaseCycle: "2"
+    releaseDate: 2023-12-01
+    eoas: 2024-12-01
+    eol: 2026-12-31
+    latest: "2.2.18"
+    latestReleaseDate: 2025-12-05
+    lts: true
+
+  - releaseCycle: "1"
+    releaseDate: 2021-12-01
+    eoas: 2023-12-01
+    eol: 2024-12-31
+    latest: "1.7.10"
+    latestReleaseDate: 2024-03-15
+    lts: true
+
+  - releaseCycle: "0"
+    releaseDate: 2020-07-01
+    eoas: 2021-12-01
+    eol: 2021-12-31
+    latest: "0.9.0"
+    latestReleaseDate: 2021-11-15
+
```

Comment:
We don't document pre-release versions (Assuming v1 was the first stable/GA release)

```suggestion
```